common/internal_model/src/test/scala/weco/catalogue/internal_model/fixtures/index/IndexFixturesBase.scala

```diff
@@ -0,0 +1,29 @@
+package weco.catalogue.internal_model.fixtures.index
+import weco.catalogue.internal_model.fixtures.elasticsearch.ElasticsearchFixtures
+import scala.io.{Codec, Source}
+import com.sksamuel.elastic4s.Index
+import weco.fixtures.Fixture
+
+trait IndexFixturesBase extends ElasticsearchFixtures with IndexFixturesE4S {
+
+  protected def getConfig(mappings: String, analysis: String) =
+    s"""{"mappings":${Source
+        .fromResource(mappings)(Codec.UTF8)
+        .mkString},
+      "settings":{"analysis":   
+      ${Source
+        .fromResource(analysis)(Codec.UTF8)
+        .mkString}
+      }}
+      """
```

Comment:
Let's extract the two calls to `Source.fromResource` as variables, so it's easier to see what the final JSON document looks like. Similar for the method on L20–27.

```suggestion
  protected def getConfig(mappings: String, analysis: String) = {
    val mappings = Source
      .fromResource(mappings)(Codec.UTF8)
      .mkString
      
    val analysis = Source
      .fromResource(analysis)(Codec.UTF8)
      .mkString
  
    s"""{"mappings": $mappings, "settings": {"analysis": $analysis}}"""
  }
```